config

```diff
@@ -0,0 +1,80 @@
+# File: config
+
+# This configuration file consists of environmental variables
+# that can be used to over ride defaults.  If you are happy
+# with the recommended defaults provided, this file can be
+# ignored.  
+#
+#If you do want to over ride defaults...
+#
+# On your staging machine, you need only worry about the
+# first two assignments- these can be modified on your staging
+# machine and then copied to be run on your target machine 
+# as individual commands before running the second curl command.
+#
+# After that, this file will be with in the repo on your target
+# machine where it needs to be edited to your satisfaction.
+# Be sure the first two variables are set to the same values
+# in the config file on your target machine as you entered on its
+# command line before running the second curl command.
+
+
+# if you wish to change either of the following two defaults,
+# edit the values on your staging machine and then copy and
+# paste the commands to be run on your target machine just
+# before running the second curl command:
+
+# PARENT_DIR designates the directory into which the repo will be
+# cloned.  The default is the current user's home directory.
+# If changed, be sure you've created the directory and set 
+# permissions appropriately.
+export PARENT_DIR=$HOME    # Used in repo.sh
+
+# piInfoDIR designates the file system of the repo itself.
+# It will be placed within the PARENT_DIR directory.
+export piInfoDIR="piInfo"  # Used in repo.sh
+
+# Don't change the next line.
+export REPO=${PARENT_DIR}/${piInfoDIR}
```

Comment:
If this isn't meant to be configurable, since it is derived from the other variables, maybe we should move it into a separate file?